helpers/internal.psm1

```diff
@@ -733,7 +766,21 @@ Function Write-RecommendationsToLogFile {
                         "Verify NIC RDMA provisioning and traffic class settings. Consider confirming NIC firmware and drivers, as well. If the problem persists, consider checking NIC cabling or physical interlinks." | Out-File $LogFile -Append -Encoding utf8 -Width 2000
                     }
                 }
+                'Stage7' {
+                    if ($NetStackResults.Failures.Stage7.PSObject.Properties.Name -contains "InterfaceFailures") {
+                        "Interface Failure Recommendations" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+                        "RDMA Perf VMSwitch Stress failed for the following target NICs:" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+                        $NetStackResults.Failures.Stage7.InterfaceFailures | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+                        "Verify NIC RDMA provisioning and traffic class settings. Consider confirming NIC firmware and drivers, as well. If the problem persists, consider checking NIC cabling or physical interlinks." | Out-File $LogFile -Append -Encoding utf8 -Width 2000

```

Comment:
"Consider confirming" - State the imperative. "Confirm firmware and drivers..."

Also, there really isn't value in "If the problem persists..." this issue is transient by nature. So it will stop as the traffic dies down even if the problem persists. When the traffic increases, the issue will manifest again == cluster down.